src/pages/edge/adobe-journey-optimizer/push-notification/android/push-to-iam.md

```diff
@@ -19,7 +19,8 @@ This document describes the steps that allow a designated in-app message to be s
 
 ## Pre-requisites
 
-[Integrate and register Messaging extension](../../../index.md#implement-extension-in-mobile-app) in your app.
+- [Integrate and register Messaging extension](../../../index.md#implement-extension-in-mobile-app) in your app.
+- Version **3.3.0** or newer of Messaging extension for Android.
```

Comment:
This way of formatting seems to cause lint errors. Should we add an `<InlineAlert variant="info" slots="text"/>` to highlight the min version?